src/ncp/posix/netif.cpp

```diff
@@ -114,6 +114,65 @@ void Netif::UpdateIp6UnicastAddresses(const std::vector<Ip6AddressInfo> &aAddrIn
     mIp6UnicastAddresses.assign(aAddrInfos.begin(), aAddrInfos.end());
 }
 
+otbrError Netif::UpdateIp6MulticastAddresses(const std::vector<Ip6Address> &aAddrs)
+{
+    otbrError error = OTBR_ERROR_NONE;
+
+    // Remove stale addresses
+    for (const Ip6Address &address : mIp6MulticastAddresses)
+    {
+        if (std::find(aAddrs.begin(), aAddrs.end(), address) == aAddrs.end())
+        {
+            otbrLogInfo("Remove address: %s", Ip6Address(address).ToString().c_str());
+            SuccessOrExit(error = ProcessMulticastAddressChange(address, false));
```

Comment:
Add comment for boolean arg? 
```suggestion
            SuccessOrExit(error = ProcessMulticastAddressChange(address, /* aIsAdded */ false));
```